### ISSUE TITLE
Website: skip to content on left hand navigation positioning issue

### ISSUE THREAD

**Issue by GCHQ-Developer-465** [mi6/ic-design-system#144]
## Summary of the bug
The focus around the skip to content is overlaying the top nav. 

## 🧐 Expected behaviour
Enough padding should be provided around it so that the focus indicator doesn't overlay anything else

## 📸 Screenshots or code
![Screenshot 2022-12-14 at 14 17 51](https://user-images.githubusercontent.com/111882114/207619976-2bd06a5c-ba6e-49ae-9ad6-2c0046c191aa.png)

**Comment by GCHQ-Developer-847** [comment]
This is no longer an issue as the 'Skip to page content' link now uses an ic-link component - so there is no longer a focus indicator:
![Image](https://user-images.githubusercontent.com/111882109/213665555-1d4c5b12-aa69-4fe3-a62e-b0e0db656159.png)
